python/example_code/lookoutvision/models.py

```diff
@@ -0,0 +1,230 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Amazon lookout for vision model code examples used in the service documentation:
+https://docs.aws.amazon.com/lookout-for-vision/latest/developer-guide/model.html
+Shows how to create and delete a model. Also, how to view the versions of the models
+in a project.
+"""
+
+import time
+import logging
+import json
+
+from botocore.exceptions import ClientError
+
+
+logger = logging.getLogger(__name__)
+
+
+class Models:
+    """
+    Provides example methods that create and manage Amazon Lookout for Vision
+    models.
+    """
+
+    @staticmethod
+    def create_model(
+        lookoutvision_client,
+        project_name,
+        training_results,
+        tag_key = None,
+        tag_key_value = None
+    ):
+        """
+        Creates a version of an Amazon Lookout for Vision model.
+        param: lookoutvision_client: The Amazon Lookout for Vision boto 3 client.
+        param: project_name: The name of the project in which you want to create a model.
+        param: training_results: The S3 location where training results are stored.
+        param: tag_key: The key for a tag to add to the model.
+        param: tag_key_value - A value associated with the tag_key.
+        return: The model status and version.
+        """
+
+        try:
+            # Create a model
+            logger.info("Training model...")
+
+            # Define training output location
+            output_bucket, output_folder = training_results.replace("s3://", "").split(
+            "/", 1 )
+
+            output_config = json.loads(
+                '{ "S3Location": { "Bucket": "'
+                + output_bucket
+                + '", "Prefix": "'
+                + output_folder
+                + '" } } '
+            )
+
+            #Add tag, if defined.
+            tags=[]
+
+            if tag_key is not None:
+                tags = json.loads(
+                '[{"Key": "' + tag_key + '" ,"Value":"' + tag_key_value + '"}]'
+            )
```

Comment:
Don't use json.loads() to define Python dict.

Reply:
Fixed.